ui/components/Footer.tsx

```diff
@@ -37,23 +37,54 @@ function Footer({ className }: Props) {
     ? `${REPO_URL}/commit/${versionData.commit}`
     : `${REPO_URL}/releases/tag/v${p.version}`;
 
+  const fluxVersionText = versionData.fluxVersion;
+  const fluxVersion =
+    !isLoading && fluxVersionText !== "" ? (
+      <>
+        <Text semiBold>Flux:</Text>
+        <Spacer padding="xxs" />
+        <Text>{fluxVersionText}</Text>
+        <Spacer padding="xxs" />
+      </>
+    ) : null;
+
+  const kubeVersionText = versionData.kubeVersion;
+  const kubeVersion =
```

Comment:
fluxVersion and kubeVersion could potentially be combined into one component with props but I don't mind either way

Reply:
Added the Version component with snapshot tests.

1) Not sure if adding `attrs` when exported is still required, like recommended in the style.md doc, so still added it like that:

```
export default styled(Version).attrs({ className: Version.name })``;
```

2) Footer versions are wrapped at narrow screens (see layout at 768 or 1000 px for example) , otherwise we would have to add a horiz scrollbar, preferably for the whole layout (or it looks ugly if added only to the footer).

Does this look OK?

<img width="735" alt="Screenshot 2022-06-28 at 08 38 53" src="https://user-images.githubusercontent.com/8824708/176110484-19bad6fe-e43e-49cb-a810-42027bc45749.png">

Reply:
Three options that come to mind:
1. Set everything to nowrap and just have it overflow the container 
2. Increase the minWidth of the container so we know we're always fitting the whole footer in
3. Decrease font size on the footer?
Maybe we do a mix of all three?

Reply:
Used options 1 and 2: set the Version component text to nowrap and changed 768px in Layout.tsx from 768px to 900px (conveniently, a Material-UI breakpoints).

Everything I tried with scrollbars in the footer and footer containers looked ugly. So, I've decided not to add `overflow-x: auto`.

I think it's either fixes or we'll need to serious redesign the footer (for example, move it under the whole app content or move content info + email text somewhere from the footer), because its current text is really wide.

So, the current version looks best for now.